pkg/config/config.go

```diff
@@ -86,7 +87,7 @@ type Config struct {
 	// CONFIG FILE
 	Version             string
 	Verbose             bool
-	Format              string
+	Format              output_formats.OutputFormat
```

Comment:
if you have output_format but then call a variable OutputFormat there is something wrong

Reply:
Is there a name you would prefer for this type? I cannot think of one better representing a type of output supported by editorconfig-checker.

Reply:
don't really have an idea if you would like to still see a change here, it being `outputformat.OutputFormat` now.

Reply:
I don't have either